muon-spectroscopy-computational-project/muon-galaxy
# Add crystvis features #12

Closes #12 
This adds support for relevant crystvis features through the Galaxy UI.
![image](https://user-images.githubusercontent.com/61705287/204827229-05c5fe50-98d1-4242-b414-3022fc927104.png)

Note that the export screenshot will only output a blank image.
**EDIT:** Having looked into this some more, integrating with the existing Galaxy screenshot buttons would require a change to Crystvis, to allow us to provide generate a renderer with `WebGLRenderer({preserveDrawingBuffer: true})`. There may be some further tinkering required in Galaxy, but this would be the first step.

## How to test the changes?
(Select all options that apply)
  1. Login
  2. Open visualisations
  3. Select crystvis, and chose a cell or cif file
  4. Test features work as expected, and labels make sense

## License
